### PR TITLE
Wiremill Circuit Recipes

### PR DESCRIPTION
**Pull request by ReclipseTheOne**
Nothing should be broken (copium)

### CODE REVIEW

**Review by mikerooni — APPROVED** [review]

**Comment by mikerooni** [comment]
Looks good so far, should be able to include this in the next release after 1.1.4

File: src/main/java/com/gregtechceu/gtceu/data/recipe/generated/WireRecipeHandler.java [hunk 1]
```diff
@@ -77,11 +83,54 @@ public static void processWireSingle(TagPrefix wirePrefix, Material material, Wi
...
+            .duration((int) material.getMass() * 8)
+            .EUt(getVoltageMultiplier(material))
+            .save(provider);
+
+        if (material.hasFlag(GENERATE_FINE_WIRE)) {
+            WIREMILL_RECIPES.recipeBuilder("mill_" + material.getName() + "_wire_fine")
+                .inputItems(prefix, material, 1)
+                .circuitMeta(3)
+                .outputItems(wireFine, material, 8)
+                .duration((int) material.getMass() * 3)
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/data/recipe/generated/WireRecipeHandler.java]: 1.12 had a loop for this yk?

**Review by screret — CHANGES_REQUESTED** [review]

File: src/main/java/com/gregtechceu/gtceu/data/recipe/generated/WireRecipeHandler.java (hunk 1 above)
Comment (screret) [reply src/main/java/com/gregtechceu/gtceu/data/recipe/generated/WireRecipeHandler.java]: would be cleaner :)

Comment (screret) [reply src/main/java/com/gregtechceu/gtceu/data/recipe/generated/WireRecipeHandler.java]: https://github.com/GregTechCEu/GregTech/blob/master/src/main/java/gregtech/loaders/recipe/handlers/WireRecipeHandler.java#L86-L95

**Review by screret — APPROVED** [review]

**Pull request merged**